test/phone.spec.ts

```diff
@@ -1,47 +1,128 @@
-import { describe, expect, it, vi } from 'vitest';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
```

Comment:
This should not be required anymore :thinking: 

```suggestion
import { beforeEach, describe, expect, it } from 'vitest';
```